build.sbt

```diff
@@ -28,7 +28,8 @@ libraryDependencies += "com.fasterxml.jackson.core" % "jackson-annotations" % "2
 libraryDependencies += "com.fasterxml.jackson.core" % "jackson-core" % "2.12.1"
 libraryDependencies += "com.softwaremill.sttp.client3" %% "core" % "3.0.0"
 libraryDependencies += "com.ibm.icu" % "icu4j" % "68.2"
-
+libraryDependencies += "org.apache.commons" % "commons-csv" % "1.8"
+libraryDependencies += "com.lihaoyi" %% "os-lib" % "0.7.7"
```

Comment:
Whats `os-lib` for?

Reply:
This was added while experimenting with the paths issue we had, didn't remove after it. I got the code to successfully compile without it, so removing it for now.